src/common/components/ui/WizardNavItem.tsx

```diff
@@ -9,7 +9,7 @@ const getNavItemContent = (
   isDisabled?: boolean,
   isCurrent?: boolean,
 ): ReactNode => {
-  if (!isDisabled && !isCurrent && !isValid()) {
+  if (!isDisabled && (!isCurrent || (isCurrent && content === 'Storage')) && !isValid()) {
```

Comment:
I don't understand this part. Isn't this about having the warning icon in the step name?

Reply:
Yes, is about the warning icon. In storage step is not shown (only in host discovery step) and is so confusing for the users

Reply:
We need to solve the problem about the user being stuck in one step and not being able to proceed to the next ones where the issue can be fixed, as is the current case.

Even if the warning icon appears in Storage, they cannot move to that step. I think it we fix the original issue this change won't be necessary.

Reply:
True. I've changed the way to retrieve the correct step depending of validations